Ginger/GingerCore/ALM/JIRA/Bll/JiraImportManager.cs

```diff
@@ -214,6 +219,204 @@ internal BusinessFlow ConvertJiraTestSetToBF(JiraTestSet testSet)
             }
         }
 
+        internal BusinessFlow ConvertJiraZypherCycleToBF(JiraZephyrCycle cycle)
+        {
+            try
+            {
+                if (cycle == null) return null;
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)